### ISSUE TITLE
Used property "fetch" on versionless client

### ISSUE THREAD

**Issue by dnlmzw** [BretCameron/sanity-plugin-order-documents#13]
I don't know if this would result in an error but I'm not able to "Choose a type", so this might be the cause:

```
Error: Used property "fetch" on versionless client - this is deprecated. Please specify API version using `withConfig` - see https://docs.sanity.io/help/studio-client-specify-api-version
    at Object.get (client:87)
    at _callee3$ (OrderDocuments.js:219)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:294)
    at Generator.next (runtime.js:119)
    at asyncGeneratorStep (asyncToGenerator.js:3)
    at _next (asyncToGenerator.js:25)
```

![not-working](https://user-images.githubusercontent.com/960055/130426162-14086774-a6d5-467d-83fb-d429680d4ef6.gif)

**Comment by dnlmzw** [comment]
Never mind. It fixed itself somehow? 🤷🏻‍♂️

**Comment by Sigve84** [comment]
Same issue here as well. I  think we need to add the version number to the fetch commands.

**Comment by BretCameron** [comment]
It seems like Sanity now requires an API version to be specified. (Before, it assumed version `"1"` as the default).

I have added this fix in version `0.0.22`.
